tests/unit/test_attribute_configuration_collection.py

```diff
@@ -0,0 +1,332 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.models.attribute_configuration.collection import (
+    AttributeConfigurationCollection,
+)
+
+
+class TestAttributeConfigurationCollection(TestCase):
+    def setUp(self):
+        auth = UsernamePasswordAuth("username", "password")
+        self.unify = Client(auth)
+
+    @responses.activate
+    def test_by_relative_id(self):
+        ac_url = f"http://localhost:9100/api/versioned/v1/projects/1/attributeConfigurations/1"
+        alias = "projects/1/attributeConfigurations/"
+        ac_test = AttributeConfigurationCollection(self.unify, alias)
+        expected = self.acc_json[0]["relativeId"]
+        responses.add(responses.GET, ac_url, json=self.acc_json[0])
+        self.assertEqual(
+            expected,
+            ac_test.by_relative_id("projects/1/attributeConfigurations/1").relative_id,
+        )
+
+    @responses.activate
+    def test_by_resource_id(self):
```

Comment:
The test in here is for relative id instead of resource id

Reply:
it checks that by_resource_id is correct by comparing relative id's

Reply:
Sorry, I misread that